index.bs

```diff
@@ -1763,7 +1763,8 @@ Initiating Charter Refinement</h3>
 	An [=Advisory Committee representative=] <em class=rfc2119>may</em> formally request
 	that the [=Team=] initiate [=charter refinement=].
 	The Team <em class=rfc2119>may</em> deny such a request
-	if it thinks the proposal is insufficiently mature or does not align with W3C's scope and mission,
+	if it thinks the proposal does not meet the charter assessment criteria
```

Comment:
```suggestion
        if it thinks the proposal is insufficiently mature,
        does not align with W3C's scope and mission,
	or otherwise thinks the proposal does not meet the charter assessment criteria
```

Here's a slightly less minimalist version, which still defers the details to the guide, but keeps the high level principles in the Process.

Reply:
Are there charter assessment criteria that don't fit under either of the other two requirements? CC @ianbjacobs @plehegar

Reply:
@fantasai,

Yes. @plehegar and I came up with these others to start:

* Does the charter align with established Web architecture principles?
* Is there sufficient support for the charter?
* Is there significant consensus for the charter? One example of lack of consensus is when there are two competing charters and the community has not yet converged.
* Are there clear signals within the W3C community that the W3C Membership will participate in the work, or that organizations will join to do the work?
* Does the Team have adequate resources for the group in light of current W3C priorities?

We think it is useful to have a list (for predictability), that we will want to make it easy for the list to evolve with practice, and that a long list should not weigh down the process doc.